### ISSUE TITLE
Non-violent communication / strategic workplace conflict resolution training?

### ISSUE THREAD

**Issue by benjamingr** [openjs-foundation/cross-project-council#1106]
Hey!

In the past the Node.js moderation team went through Conflict Management Training offered by the Node.js foundation through Rockdale College. I believe @hackygolucky was the organizer of that around ±5 years ago.

I think it was useful and it would be beneficial to offer this training again to the projects' moderation teams (and possibly the OpenJS org moderation team when/if it gets established) as a lot of people have joined (or left) since then.

The training provided useful de-escalation techniques, tips on how to be more empathetic in issues and tools for managing constructive conflict.

I'm just posting this here to bring it up, if there are already initiatives/efforts in this area that I'm unaware of then apologies for my poor research skills :)

Happy to also provide information about the training last time in case that's useful.

**Comment by benjamingr** [comment]
I'll ping the Node.js members @Trott (who attended the training last time and may have feedback) and @joesepi as well as voting members @eemeli @tobie and @mhdawson @ljharb and @rginn as the director.

I have _no idea_ if this visibility/pings is useful so by all means if I tagged someone I shouldn't have or there is a process I'm doing wrong please call me out on it so I can learn.

**Comment by ovflowd** [comment]
Also, +1 from my side, I believe we need to be better when communicating and bringing bias.

**Comment by mhdawson** [comment]
+1 from me as well. It would be good to have that available on a regular basis for those who are interested.

**Comment by rginn** [comment]
I'm attending diversity workshops for Linux Foundation staff this month led by Dr. Gloria Chance. https://www.linkedin.com/in/drgloriachance/. Topics include Psychological Safety and Being an Ally. I'll report back on the trainings and explore options for our OpenJS communities.

**Comment by hackygolucky** [comment]
oh hi @benjamingr! I couldn't find the vendor name in my notes so thanks for that. The structure was Zoom meetings and sometimes breakout Zoom rooms, with some homework as part of the training. It was a great practice for folks on the Moderation team for sure, and I still value what they walked us through and use it in management and my day to day. It's especially great for anyone in a perceived position of power(such as the CPC)

**Comment by tobie** [comment]
Both of these sound great.

**Comment by benjamingr** [comment]
> I've found self-paced courses to be worth little more than helping meet compliance requirements for this kind of topic. imho, this tends to be the kind of topic where in person training is much more effective (whether onsite or online). 

I agree with this sentiment and I thought the fact the course was "live" (on zoom) with real instructors and exercises was important.

**Comment by Relequestual** [comment]
Very interested in this!
Also need to find people to form our CoC review committee.

**Comment by tobie** [comment]
> Also need to find people to form our CoC review committee.

FYI, @Relequestual, you can chose to delegate this work to the CPC, either temporarily or permanently.

**Comment by wesleytodd** [comment]
If there was a way to make this available to a larger group I think it would be awesome to see if any other projects leadership under the foundation wanted to attend.

**Comment by benjamingr** [comment]
Any update on this?

**Comment by ctcpip** [comment]
In another group, we reached out to some NVC training providers, and I think we can share what that entailed (curriculum/format) and cost.  Though cost now is probably out-of-date, this may help with planning and exploring options.

**Comment by ovflowd** [comment]
@benjamingr We're waiting on the staff, so it hasn't stalled... I bet they're doing something behind the scenes, but since the last update was in April. cc @benjamingr 

Apologies for it taking so long!

**Comment by benjamingr** [comment]
Naa, this issue is about a year old for something that should take about an hour or two to set up.

This isn't criticism of OpenJS as a whole, it's just not clear the foundation is super into this. This doesn't have a champion inside or the priority is just low or people just don't think it's important. That's fine, nothing in the OpenJS foundation governance requires them to provide this sort of training or to proactively push for it. In other areas where we needed the foundation (like legal questions) they helped Node and provide a useful and important legal framework.

So rather than try a fourth ping here which I suspect will just bug people - I think it's fine to ask for funding and set this up at the Node.js project level and then if successful champion it at the OpenJS level.

**Comment by tobie** [comment]
Alternative option: show up at the CPC calls and champion it.

**Comment by benjamingr** [comment]
> Alternative option: show up at the CPC calls and champion it.

How? Can I just show up to a call and add an item to the agenda?

**Comment by tobie** [comment]
> How? Can I just show up to a call and add an item to the agenda?

Pretty much. Anything with the [`waiting-on-staff-update`](https://github.com/openjs-foundation/cross-project-council/labels/waiting-on-staff-update) label get generally reviewed at the beginning of the call, but I've added the [`cross-project-council-agenda`](https://github.com/openjs-foundation/cross-project-council/labels/cross-project-council-agenda) label to make sure it would be discussed next Tuesday.

**Comment by rginn** [comment]
I've connected with a vendor and am awaiting a budget proposal.

**Comment by joesepi** [comment]
Meeting notes:
  * removing agenda label as it is captured by previous label (waiting on updates)

**Comment by benjamingr** [comment]
@rginn any update?

**Comment by benjamingr** [comment]
Awesome thanks for that I really appreciate it Robin and happy we'll get that training.

For Node I think it would be super useful for the moderation team and TSC to attend.

**Comment by tobie** [comment]
**Notes from today's CPC call:**

@rginn informed the CPC that she had reached back to the trainer who was on leave until Oct. 7 and would update the CPC when she would hear back.

**Comment by tobie** [comment]
**Notes from this week's CPC call:**

@rginn has a meeting planned end of Novemeber and will update us after the call.

**Comment by rginn** [comment]
We’ve secured the conflict management consulting firm Insight Partners https://insightpartnersonline.com/ for 2 half-day training sessions with Elise Willeau and David Seibel. Thanks @wesleytodd for the referral. 

We would like to schedule the sessions for March or April and selected Thursday/Friday options based on CPC feedback. Please indicate which two days work best for you to attend. Participation is for active community members of the OpenJS Foundation and our hosted projects. We’ll select the month/date that works for the majority of interested people.

**March 6 & 7  (11- 3 EST) (indicate with tada / celebration emoji) -- or --
April 17 & 18 (11-3 EST) (indicate with rocket emoji)** 

**General Purposes.  The core objectives of these workshops are to:**

- Increased alignment on the mindsets and skills necessary to develop, preserve and repair relationships through efficient and effective communication. 
- Increase awareness of the visible and hidden patterns typical of tough conversations.
- Introduce a systematic framework and tools for handling and preparing for conversations participants find tough.  
- Give participants opportunities to assess and improve their skills.  
- Help participants connect their learning to their professional and personal challenges going forward.  

**Outcomes.  Primary benefits to participants include:**

- Increased awareness of the dynamics typical of the challenging conversations relevant to participants. 
- Structured frameworks and tools for preparing and handling difficult conversations.
- Best practice advice, including what to do with right/wrong thinking, blame, unintended negative impact, and strong emotions in others and ourselves.
- Practice applying frameworks and best practice advice to real-world challenges.
- Opportunities to build relationships and share best practice advice among participants.

**Comment by ljharb** [comment]
There's a TC39 plenary April 14th - 17th, so anyone with an overlap wouldn't be able to make the first day.

**Comment by UlisesGascon** [comment]
Aprils seems tricky:
- [Node Congress](https://nodecongress.com/) event (17th-18th)
- That week include many holidays several many countries ([Spain](https://www.timeanddate.com/holidays/spain/), [UK](https://www.timeanddate.com/holidays/uk/), [Germany](https://www.timeanddate.com/holidays/germany/https://www.timeanddate.com/holidays/germany/), [France](https://www.timeanddate.com/holidays/france/)... )

**Comment by tobie** [comment]
CPC call: Current proposal to run the sessions March 6 & 7 for CPC members and lead maintainers. Registration form to follow.

**Comment by rginn** [comment]
The workshop is confirmed for March 6 & 7 (11- 3 EST). We will be sending out registration forms to our maintainers and Board members.

**Comment by rginn** [comment]
I emailed a calendar invite and an [RSVP form](https://docs.google.com/forms/d/e/1FAIpQLScjC3H_0F-5Ka3y1WLz39zZ7LIzBZBA5s9toNg0ZEkIsl66Jw/viewform) for OpenJS Maintainers and Board members. 

Please DM me if you need the invite / zoom info.
